lib/cf_obs_binary_builder/obs_package.rb

```diff
@@ -36,7 +36,12 @@ def checkout(config = {}, &block)
         Dir.chdir(name) do
           if config[:reset]
             log 'Resetting package'
-            FileUtils.rm Dir.glob("*")
+            if config[:keep_patches]
```

Comment:
I would prefer if the patch comes out of cf-obs-binary-builder so we do not need a special handling and can also remove it if it is obsolete.

Reply:
Or maybe even a sed statement is possible if it is a simple patch.

Reply:
This should be fixed now, patches are under ``templates/patches/``